### PR TITLE
Certificate Travel Infos wrong text

### PR DESCRIPTION
**Pull request by CV113**
Certificate Travel Infos wrong text; replacing the word "Wählen" with "Geben"

Jira Ticket: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11228

### CODE REVIEW

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -2811,7 +2811,7 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "HealthCertificate_Validation_Info_title" = "Informationen";
 
-"HealthCertificate_Validation_Info_byCar" = "Wählen Sie die geplante lokale Ankunftszeit an der Grenze zu Ihrem Einreiseland aus.";
+"HealthCertificate_Validation_Info_byCar" = "Geben Sie die geplante lokale Ankunftszeit an der Grenze zu Ihrem Einreiseland aus.";
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2814]:
```suggestion
"HealthCertificate_Validation_Info_byCar" = "Geben Sie die geplante lokale Ankunftszeit an der Grenze zu Ihrem Einreiseland ein.";
```

**Comment by Ein-Tim** [comment]
This PR reverts https://github.com/corona-warn-app/cwa-app-ios/pull/3707.

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Comment by naveeddotio** [comment]
> This PR partly reverts #3707.
> 
> Why? The info screen and the note wouldn't be in sync anymore, if this is merged.

@CV113

**Pull request merged**